pandas/tests/series/test_timeseries.py

```diff
@@ -373,6 +373,24 @@ def test_pct_change(self):
             rs, (filled / filled.shift(freq="5D") - 1).reindex_like(filled)
         )
 
+    @pytest.mark.parametrize("n_1", ["1", "15"])
+    @pytest.mark.parametrize("offset_1", ["D", "BM"])
```

Comment:
can you add a test with the original example exactly, I really can't tell if this test works as expected.

Reply:
okay, no problem. I have added `empty n` which is equivalent to 1 and size 70 for assuring original example(DataFrame with `70D` and pct_change with `BM`).

Reply:
Currently, Travis CI gives me this error repeatedly:
```
Install Miniconda
PREFIX=/home/travis/miniconda3
Unpacking payload ...
  0%|                                                    | 0/35 [00:00<?, ?it/s]
No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.
Check the details on how to adjust your build configuration on: https://docs.travis-ci.com/user/common-build-problems/#build-times-out-because-no-output-was-received
The build has been terminated
```
Can you retrigger CI after it recovers? I think I don't have a right to retrigger it...